src/components/telemetry_monitor/CMakeLists.txt

```diff
@@ -69,6 +71,14 @@ add_library("TelemetryMonitor" ${SOURCES})
 target_link_libraries("TelemetryMonitor" ${LIBRARIES})
 add_dependencies("TelemetryMonitor" HMI_API MOBILE_API)
 
+set (LIBRARIES
```

Comment:
@dev-gh Right but these changes are critical and needed for further development.